core/src/world/place/building/religious/shrine.rs

```diff
@@ -11,11 +12,11 @@ fn name(rng: &mut impl Rng) -> String {
         0..=3 => format!("The {} {}", descriptor(rng), place(rng)),
         4..=7 => format!("{} of {}", place(rng), deity(rng)),
         8 => {
-            let (animal, s) = pluralize(animal(rng));
+            let (animal, s) = pluralize(any_animal(rng));
```

Comment:
Not a problem for today, but the ergonomics here are starting to bother me. I'd expect this operation to look more like:

```suggestion
            let (animal, s) = any_animal(rng).plural();
```

But that would require some messing with traits that can be dealt with another day. See `crate::utils::case_insensitive_str` for an example of how the implementation might look.